### ISSUE TITLE
GitHub Action: Add extend-disable input for disabling rules without a config file

### ISSUE THREAD

**Issue by matcra587** [rvben/rumdl#406]
## Summary

Add an `extend-disable` input to the GitHub Action so rules can be disabled without requiring a config file in every consuming repository.

## Motivation

When using `rumdl` via reusable workflows (`workflow_call`), the calling repo currently needs a `.rumdl.toml` just to disable a rule like `MD013`. This creates config file sprawl across many repos for what is often a single-line override. The CLI already supports `--extend-disable` — exposing it as an action input would let reusable workflows pass it through cleanly.

## Proposed behavior

```yaml
- uses: rvben/rumdl@v0.1.7
  with:
    extend-disable: "MD013,MD024"
```

Maps to `--extend-disable MD013,MD024` on the CLI. Comma-separated list, consistent with the existing `--extend-disable` flag.

### Alternative: generic `args` input

```yaml
- uses: rvben/rumdl@v0.1.7
  with:
    args: "--extend-disable MD013"
```

More flexible but harder to validate and could conflict with existing inputs (e.g. `--output-format` vs `report-type`). A specific `extend-disable` input is safer and more discoverable.

**Comment by rvben** [comment]
Thanks for the request, @matcra587. Implemented in v0.1.21 — the GitHub Action now has a generic `args` input for passing extra CLI flags:

```yaml
- uses: rvben/rumdl@v0.1.21
  with:
    args: '--output-format json --disable MD013,MD033'
```

This covers the `extend-disable` use case and any other CLI flags without needing a dedicated input for each.
